pages/about.html

```diff
@@ -1,12 +1,74 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../styles/about.css">
+    <title>Document</title>
 </head>
+
 <body>
-  
+    <nav class="navbar">
+        <div class="container">
+            <div class="logo">ABOUT</div>
+            <ul class="nav">
+                <li>
+                    <a href="index.html">Home</a>
+                </li>
+                <li>
+                    <a href="portfolio.html">portfolio</a>
+                </li>
+            </ul>
+        </div>
+    </nav>
+    <header class="header ">
+        <div class="container ">
+            <div>
+                <h1>Education & Work Experience</h1>
+                <p>About My Education and Work Experiences</p>
+            </div>
+            <img src="../image/vida.png" alt="Vida Maleki">
+        </div>
+    </header>
+    <section class="boxes ">
+        <div class="container ">
+            <div class="box ">
+                <h2>University</h2>
+                <p>Bachelor's degree in accounting</p>
+                <p>Islamic Azad University of Tabriz</p>
+                <p>From 2009 to 2014</p>
+            </div>
+            <div class="box ">
+                <h2>YWCA</h2>
+                <p>Bank$Work certificate in 2018</p>
```

Comment:
In semantic HTML, heading tags should be used to indicate different sections of content, not as stylistic elements. Start from the highest level first (`h1`) and then work your way down. Adjust styling and sizing using CSS if needed.  [Link](https://bighack.org/how-to-heading-structure-accessibility/)